.coveragerc

```diff
@@ -0,0 +1,348 @@
+    *sqlalchemy/events.py*
+    *sqlalchemy/exc.py*
+    *sqlalchemy/ext/__init__.py *
+    *sqlalchemy/ext/compiler.py*
+    *sqlalchemy/ext/declarative/__init__.py *
+    *sqlalchemy/ext/declarative/api.py   *
+    *sqlalchemy/ext/declarative/base.py *
+    *sqlalchemy/ext/declarative/clsregistry.py*
+    *sqlalchemy/inspection.py *
+    *sqlalchemy/interfaces.py *
+    *sqlalchemy/log.py *
+    *sqlalchemy/orm/__init__.py*
+    *sqlalchemy/orm/attributes.py*
+    *sqlalchemy/orm/base.py  *
+    *sqlalchemy/orm/collections.py *
+    *sqlalchemy/orm/dependency.py*
+    *sqlalchemy/orm/deprecated_interfaces.py   *
+    *sqlalchemy/orm/descriptor_props.py *
+    *sqlalchemy/orm/dynamic.py *
+    *sqlalchemy/orm/evaluator.py   *
+    *sqlalchemy/orm/events.py*
+    *sqlalchemy/orm/exc.py *
+    *sqlalchemy/orm/identity.py  *
+    *sqlalchemy/orm/instrumentation.py*
+    *sqlalchemy/orm/interfaces.py*
+    *sqlalchemy/orm/loading.py   *
+    *sqlalchemy/orm/mapper.py*
+    *sqlalchemy/orm/path_registry.py*
+    *sqlalchemy/orm/persistence.py *
+    *sqlalchemy/orm/properties.py  *
+    *sqlalchemy/orm/query.py *
+    *sqlalchemy/orm/relationships.py *
+    *sqlalchemy/orm/scoping.py*
+    *sqlalchemy/orm/session.py   *
+    *sqlalchemy/orm/state.py*
+    *sqlalchemy/orm/strategies.py*
+    *sqlalchemy/orm/strategy_options.py *
+    *sqlalchemy/orm/sync.py*
+    *sqlalchemy/orm/unitofwork.py*
+    *sqlalchemy/orm/util.py *
+    *sqlalchemy/pool.py*
+    *sqlalchemy/processors.py *
+    *sqlalchemy/schema.py  *
+    *sqlalchemy/sql/__init__.py*
+    *sqlalchemy/sql/annotation.py  *
+    *sqlalchemy/sql/base.py *
+    *sqlalchemy/sql/compiler.py*
+    *sqlalchemy/sql/crud.py *
+    *sqlalchemy/sql/ddl.py*
+    *sqlalchemy/sql/default_comparator.py*
+    *sqlalchemy/sql/dml.py   *
+    *sqlalchemy/sql/elements.py*
+    *sqlalchemy/sql/expression.py  *
+    *sqlalchemy/sql/functions.py *
+    *sqlalchemy/sql/naming.py *
+    *sqlalchemy/sql/operators.py *
+    *sqlalchemy/sql/schema.py*
+    *sqlalchemy/sql/selectable.py*
+    *sqlalchemy/sql/sqltypes.py  *
+    *sqlalchemy/sql/type_api.py  *
+    *sqlalchemy/sql/util.py *
+    *sqlalchemy/sql/visitors.py*
+    *sqlalchemy/types.py *
+    *sqlalchemy/util/__init__.py*
+    *sqlalchemy/util/_collections.py *
+    *sqlalchemy/util/compat.py *
+    *sqlalchemy/util/deprecations.py *
+    *sqlalchemy/util/langhelpers.py*
+    *sqlalchemy/util/queue.py *
+    *sqlalchemy/util/topological.py*
+    *werkzeug/__init__.py*
+    *werkzeug/_compat.py*
+    *werkzeug/_internal.py*
+    *werkzeug/datastructures.py*
+    *werkzeug/exceptions.py*
+    *werkzeug/filesystem.py*
+    *werkzeug/formparser.py *
+    *werkzeug/http.py*
+    *werkzeug/local.py *
+    *werkzeug/routing.py *
+    *werkzeug/security.py*
+    *werkzeug/urls.py*
+    *werkzeug/utils.py *
+    *werkzeug/wrappers.py*
+    *werkzeug/wsgi.py*
+    *sqlite/__init__.py *
+    *sqlite/dbapi.py *
+    *stringprep.py *
+    *uuid.py
```

Comment:
You can omit these files in a simpler way, by adding the site packages folder, eg  `*/site-packages/*`